src/theory/theory.cpp

```diff
@@ -75,9 +75,10 @@ Theory::Theory(TheoryId id,
       d_sharedTermsIndex(satContext, 0),
       d_careGraph(nullptr),
       d_instanceName(name),
-      d_checkTime(getStatsPrefix(id) + name + "::checkTime"),
-      d_computeCareGraphTime(getStatsPrefix(id) + name
-                             + "::computeCareGraphTime"),
+      d_checkTime(smtStatisticsRegistry().registerTimer(
+          getStatsPrefix(id) + name + "::checkTime")),
+      d_computeCareGraphTime(smtStatisticsRegistry().registerTimer(
+          getStatsPrefix(id) + name + "::computeCareGraphTime")),
```

Comment:
Just to make sure: `name` does not end with `::`?

Reply:
Yes. Here, the name is also used for other stuff where we don't want the trailing `::`.